File: 2017/nazionali/classifica/gradergen/include_grader.pas
procedure make_calls(var Q: longint; types: array of char; params: array of longint; var answers: array of longint);
var
	i, cnt: longint;
begin
	cnt := 0;
	for i := 0 to Q-1 do
    begin
		answers[i] := 0;
		if types[i] = 's' then
   			supera(params[i])
		else if types[i] = 'x' then
			squalifica(params[i])
		else if types[i] = 'p' then begin
			answers[cnt] := partecipante(params[i]);
			cnt := cnt + 1;
		end;
    end;
	Q := cnt;
end;

File: 2017/nazionali/classifica/gradergen/include_grader.c
void make_calls(int* Q, char types[], int params[], int answers[]) {
    int cnt = 0;
	for (int i = 0; i < *Q; i++) {
		if (types[i] == 's') {
			supera(params[i]);
		}
		else if (types[i] == 'x') {
			squalifica(params[i]);
		}
		else if (types[i] == 'p') {
			answers[cnt++] = partecipante(params[i]);
		}
	}
    *Q = cnt;
}
